### PR TITLE
9356 - Clinic letter on case for trial

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.test.js [hunk 1]
```diff
@@ -1060,59 +1056,3 @@ describe('serveCaseToIrsInteractor', () => {
     });
   });
 });
-
-describe('addDocketEntryForPaymentStatus', () => {
```
Comment (TomElliottFlexion) [inline shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.test.js:1064]: Already moved to `serveCaseToIrsInteractor.addDocketEntryForPaymentStatus.test.js`

**Review by jimlerza — APPROVED** [review]

File: shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js [hunk 2]
```diff
@@ -29,6 +77,7 @@ const { UnauthorizedError } = require('../../../errors/errors');
  * @param {Uint8Array} deconstructed.standingPretrialPdfData the pdf data for the standing pretrial
  */
 const serveNoticesForCase = async ({
+  appendClinicLetter,
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js:81]: Nit: docs

File: shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js [hunk 3]
```diff
@@ -225,17 +325,16 @@ const setNoticeForCase = async ({
 
   caseEntity.addDocketEntry(standingPretrialDocketEntry);
 
-  const servedParties = aggregatePartiesForService(caseEntity);
-
+  // TODO: consider moving these two (four) calls to below `serveNoticesForCase`
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js]: Nit: can this be removed?

File: shared/src/business/entities/cases/Case.isPractitioner.test.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      {
+        applicationContext,
+      },
+    );
+    expect(
+      caseRecord.isPractitioner('3bcd5fb7-434e-4354-aa08-1d10846c1867'),
+    ).toEqual(true);
+  });
+
+  it('returns false is the contactId is not any form of practitioner', () => {
```
Comment (rosiehsmith) [inline shared/src/business/entities/cases/Case.isPractitioner.test.js]: Nit: wording

File: shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js [hunk 5]
```diff
@@ -29,6 +77,7 @@ const { UnauthorizedError } = require('../../../errors/errors');
  * @param {Uint8Array} deconstructed.standingPretrialPdfData the pdf data for the standing pretrial
```
Comment (rosiehsmith) [inline shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js:78]: Nit: jsdocs

File: shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js (hunk 3 above)
Comment (rosiehsmith) [inline shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js]: Is this for DOD?

**Review by rosiehsmith — APPROVED** [review]

File: shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js (hunk 3 above)
Comment (rosiehsmith) [reply shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js]: Sorry y'all i didn't see @rachelschneiderman's comments

**Pull request merged**